### PR TITLE
Feature/fr 83 create donate button

### PR DESCRIPTION
**Pull request by VladimirKarin**
Donate Button Component with style

### CODE REVIEW

File: src/components/Buttons/DonateButton.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    return (
+        <Button
+            className="submit-btn"
+            type="submit"
+            variant="contained"
+            color="primary"
+            size="large"
+            onClick={handleSubmit}
+        >
+            Submit
```
Comment (fedosejev) [inline src/components/Buttons/DonateButton.js]: Should the label say `Donate`?

File: src/components/Buttons/DonateButton.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Button } from '@mui/material';
+
+const DonateButton = ({ handleSubmit }) => {
+    return (
+        <Button
+            className="submit-btn"
```
Comment (fedosejev) [inline src/components/Buttons/DonateButton.js]: Can you change the class name to `donate-button`?

File: src/components/Buttons/DonateButton.js (hunk 1 above)
Comment (VladimirKarin) [reply src/components/Buttons/DonateButton.js]: Certainly. ) Fixed that.

File: src/components/Buttons/DonateButton.js (hunk 2 above)
Comment (VladimirKarin) [reply src/components/Buttons/DonateButton.js]: That would make more sense. Fixed that in the component,and in CSS.

**Review by fedosejev — APPROVED** [review]

**Pull request merged**